tests/pfc/test_pfc_unpause.py

```diff
@@ -0,0 +1,196 @@
+                pfc_class_0=0 if 0 in lossless_priorities else None,
+                pfc_class_1=1 if 1 in lossless_priorities else None,
+                pfc_class_2=2 if 2 in lossless_priorities else None,
+                pfc_class_3=3 if 3 in lossless_priorities else None,
+                pfc_class_4=4 if 4 in lossless_priorities else None,
+                pfc_class_5=5 if 5 in lossless_priorities else None,
+                pfc_class_6=6 if 6 in lossless_priorities else None,
+                pfc_class_7=7 if 7 in lossless_priorities else None,
+            )
+        )
+    )
+
+    rx_l1 = layer1.Layer1(
+        name='rxl1', port_names=[rx.name], speed=settings.speed, media='fiber',
+        flow_control=layer1.FlowControl(choice=layer1.Ieee8021qbb())
+    )
+
+    flows = []
+    for prio in lossless_priorities:
+        flows.append(
+            flow.Flow(
+                name='tx_prio_{}'.format(prio),
+                tx_rx=flow.TxRx(
+                    flow.PortTxRx(tx_port_name=tx.name, rx_port_name=rx.name)
+                ),
+                packet=[
+                    flow.Header(
+                        flow.Ethernet(
+                            src=flow.Pattern('00:CD:DC:CD:DC:CD'),
+                            dst=flow.Pattern('00:AB:BC:AB:BC:AB'),
+                            pfc_queue=flow.Pattern(str(prio))
+                        )
+                    ),
+                    flow.Header(
+                        flow.Ipv4(
+                            src=flow.Pattern('1.1.1.2'),
+                            dst=flow.Pattern('1.1.1.1'),
+                            priority=flow_ipv4.Priority(
+                                flow_ipv4.Dscp(flow.Pattern(str(prio)))
+                            )
+                        )
+                    )
+                ],
+                duration=flow.Duration(
+                    flow.FixedPackets(
+                        packets=packets, delay=10**9, delay_unit='nanoseconds'
+                    )
+                ),
+                size=flow.Size(size),
+                rate=flow.Rate(value=packets / 10, unit='pps')
+            )
+        )
+
+    flows.append(
+        flow.Flow(
+            name='rx_pfc_pause',
+            tx_rx=flow.TxRx(
+                flow.PortTxRx(tx_port_name=rx.name, rx_port_name=tx.name)
+            ),
+            packet=[
+                flow.Header(
+                    flow.PfcPause(
+                        src=flow.Pattern('00:AB:BC:AB:BC:AB'),
+                        class_enable_vector=flow.Pattern('FF'),
+                        control_op_code=flow.Pattern('0101'),
+                        pause_class_3=flow.Pattern('FFFF'),
+                        pause_class_4=flow.Pattern('FFFF')
+                    )
+                )
+            ],
+            duration=flow.Duration(flow.FixedSeconds(seconds=10)),
+            size=flow.Size(size),
+            rate=flow.Rate(value=packets / 10, unit='pps')
```

Comment:
let's use 50% of line rate here, other 50% will be shared by unpause flow when its time comes.

Reply:
changed to 50%